python/tests/test_node.py

```
@@ -0,0 +1,490 @@
+import pytest
+import asyncio
+from unittest.mock import Mock, AsyncMock
```

Ruff (0.8.2)

`unittest.mock.Mock` imported but unused

Remove unused import: `unittest.mock.Mock`

(F401)